lib/engine.py

```diff
@@ -104,6 +104,7 @@ def source_gen():
         pos = 0
         for line in real_lines:
             if not self.table.fixed_width:
+                line = line.replace('\n', '')
```

Comment:
Is this dealing with mid-line newline characters? If so we should add a comment about why. If not then I don't understand what this line is doing that the next line doesn't accomplish.